### DISCUSSION TITLE
Customizing DateTime Parsing

### DISCUSSION THREAD

**Discussion by ecwyne** [prisma/prisma#13675] [Q&A]
I'm reading from a Microsoft SQL Server that has Dates stored in a non-ISO format.

Eg. `2022-06-14 00:00:00.000` (this would actually be `2022-06-14T07:00:00.000Z`)

How can I control how this DateTime is parsed in Prisma Client?

**Comment by austincrim** [comment]
Hey @ecwyne 👋,

The Prisma Client should return all dates in ISO format, no matter the provider. 

Are you interested in returning a different format? You could do this formatting manually in a [middleware](https://www.prisma.io/docs/concepts/components/prisma-client/middleware).

**Reply by vladi-stevanovic** [reply]
👋 Hello @ecwyne  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
